opte-api/src/ip.rs

```diff
@@ -157,6 +198,7 @@ pub enum Protocol {
     ICMP = 0x1,
     IGMP = 0x2,
     TCP = 0x6,
+    Chaos = 0x10,
```

Comment:
This sounds fun! What's this part about, actually?

Reply:
This was just to make a change to opte-api to test that the check script catches it in CI. It will be removed before putting back this change.

Reply:
Sweet, thanks!